### ISSUE TITLE
[Design][FE] - Auth Profile - Accessibility Issue - Connected Apps Featured Content Heading - H3

### ISSUE THREAD

**Issue by cgednalske** [department-of-veterans-affairs/va.gov-team#37363]
## Background

An accessibility concern surfaced via automated tool aXe when checking for accessibility issues. The tool identified a moderate issue with the current heading level in featured content of Connected apps - which differs from the other featured content sections in the rest of Profile.

Concern identified: Heading order is invalid. Heading levels should only increase by one.

<img width="671" alt="Connected apps featured content" src="https://user-images.githubusercontent.com/79372956/155031061-1ab6af11-d8a7-4600-93c7-5eb15c771cb8.png">

## Remediation Guidance

Ensure heading order is semantically correct and matches other featured content sections in Profile, which pass the aXe accessibility inspection. Connected apps currently uses h3, whereas other sections of Profile use h2 for this heading.

**Connected Apps Featured Content**

<img width="510" alt="Inspect - connected apps" src="https://user-images.githubusercontent.com/79372956/155031106-7e918722-ed26-45ef-92ba-4524f546f08d.png">

**Account Security Featured Content**

<img width="537" alt="Inspect - account security" src="https://user-images.githubusercontent.com/79372956/155031123-e95383c0-feba-4185-8bb6-281a6df205ab.png">

## Tasks
- [ ] Change the H3 to H2 (see comments for more details)
- [ ] Ensure heading levels pass accessibility testing 
- [ ] Check heading levels are inline with featured content areas in other sections of Profile

## Does this impact visual design?

Possibly - check that designs reflect proper heading levels.

## Relevant standards

_Standards in WCAG that relates to the issue or best practice highlighted._
"Headings < h2 > through < h6 > represent increasing degrees of "indentation" in our conceptual "outline". As such, it does not make sense to skip heading levels, such as from < h2 > to < h4 >, going down the page." ([WebAIM source](https://webaim.org/techniques/semanticstructure/)) ([W3C source](https://www.w3.org/WAI/tutorials/page-structure/headings/))

## Acceptance Criteria
- [x] Heading levels for featured content in Connected apps increase by only one
- [ ] Heading levels pass accessibility testing
- [ ] Heading levels are in line with featured content areas in other sections of Profile

## How to configure this issue
- Include practice area in title -- e.g., [Design], [FE], [BE]
- Include project/initiative name - e.g., Auth Profile: Address Change Messaging
- Add label for practice area (frontend, backend, design)
- Add label for project if applicable
- Assign to a Profile team member
- Associate with an Epic if applicable
- If creating for yourself: estimate work if enough info exists to do so

**Comment by andaleliz** [comment]
@SKasimow I'm not sure since I didn't write the ticket but I _think_ it should be `h3` and so I changed it to that :)

**Comment by cgednalske** [comment]
@andaleliz @SKasimow oh I wrote that and yes it was P3 for priority level 3. I created the ticket while going through some accessibility training and it said to include the priority level like that, but feel free to remove it or keep as H3 because that also applies 😄

**Comment by cgednalske** [comment]
Here's the [feedback received from Josh](https://dsva.slack.com/archives/C8E985R32/p1662751008954599?thread_ts=1662750656.998289&cid=C8E985R32) in the #accessibility-help channel. 

> Yes. Assuming "Have more questions about connected apps?" is the h3 you're referring to, bumping it up to a h2 is the appropriate fix.
> Even in the current state of h1 > h2 > h3, the way that structures will make it seem like "have more questions about..." is only relevant to the last connected app.

**Comment by jcatergovy** [comment]
ready for validation @SKasimow

**Comment by SKasimow** [comment]
@cgednalske - are you able to validate this ticket?

**Comment by cgednalske** [comment]
@SKasimow sure! I'll take a look on staging 👍

**Comment by cgednalske** [comment]
@SKasimow @jcatergovy it's looking good to me! 👍

**Comment by SKasimow** [comment]
Thanks so much @cgednalske ...closing
